llama_index/response_synthesizers/accumulate.py

```diff
@@ -1,10 +1,11 @@
 import asyncio
-from typing import Any, List, Optional, Sequence
+from typing import Any, List, Optional, Sequence, Tuple
 
 from llama_index.async_utils import run_async_tasks
 from llama_index.indices.service_context import ServiceContext
 from llama_index.prompts import BasePromptTemplate
 from llama_index.prompts.default_prompts import DEFAULT_TEXT_QA_PROMPT
+from llama_index.prompts.mixin import PromptDictType, PromptMixinType
```

Comment:
don't we need to expose the `text_qa_template` here?

Reply:
yah i haven't added this yet